R/geom-defaults.r

```diff
@@ -13,7 +13,11 @@
 #' @rdname update_defaults
 update_geom_defaults <- function(geom, new) {
   g <- check_subclass(geom, "Geom", env = parent.frame())
-  old <- g$default_aes
+
+  env <- new.env()
```

Comment:
If geoms are themeable, I think we can deprecate this function, so we should just leave as is and in the documentation mark it as internal, and point people towards the new theme system.

Reply:
If we deprecate this, we will presumably need to make **all** aesthetics themeable so as to not lose the functionality?

Reply:
Oh yeah. In that case we should pull this out into a function that can be shared with `eval_defaults()`